editor/script/engine/bitsy.js

```diff
@@ -234,9 +234,24 @@ function onready(startWithTitle) {
 		canvas.addEventListener('touchend', input.ontouchend, {passive:false});
 	}
 	else {
-		document.addEventListener('touchstart', input.ontouchstart, {passive:false});
-		document.addEventListener('touchmove', input.ontouchmove, {passive:false});
-		document.addEventListener('touchend', input.ontouchend, {passive:false});
+		// creates a 'touchTrigger' element that covers the entire screen and can universal have touch event listeners added w/o issue.
+
+		// we're checking for existing touchTriggers both at game start and end, so it's slightly redundant.
+	  	let existingTouchTrigger = document.querySelector('#touchTrigger');
```

Comment:
this is a small thing, but I've been using "var" everywhere in the bitsy codebase; I'd prefer to stick with that for consistency